internal/controller/application_controller.go

```diff
@@ -36,7 +36,10 @@ import (
 	appstudioredhatcomv1alpha1 "github.com/redhat-appstudio/application-api/api/v1alpha1"
 )
 
-const NamespaceServiceAccountName = "konflux-integration-runner"
+const (
+	NamespaceServiceAccountName = "konflux-integration-runner"
```

Comment:
Name is not clear. We can have sevaral namespace-wide Service Accounts. Please be more specific.